decred/decred/dcr/wire/msgpong.py

```diff
@@ -0,0 +1,84 @@
+"""
+Copyright (c) 2020, The Decred developers
+See LICENSE for details.
+
+Based on dcrd MsgPing.
+"""
+
+from decred.util.encode import ByteArray
+
+
+CmdPong = "pong"
+
+NonceLength = 8
+
+
+class MsgPong:
+    """
+    MsgPong implements the Message API and represents a Decred pong message
+    which is used primarily to confirm that a connection is still valid in
+    response to a Decred ping message (MsgPing).
+
+    This message was not added until protocol versions AFTER BIP0031Version.
+    """
+
+    def __init__(self, nonce):
+        """
+        Args:
+            nonce (int): Unique value associated with message that is used to
+                identify specific ping message.
```

Comment:
Unique value associated with **a** message that is used to identify **a** specific **pong**(?) message.

I don't know, but is the pong nonce the same as the ping nonce? If so I think that should be mentioned here. If not then the above should be pong.